src/sys.d.ts

```diff
@@ -0,0 +1,1352 @@
+export type JsObj = Obj | number | string | boolean | ((...args: any) => any)
+
+export class Obj {
```

Comment:
Missing docs, these classes and their fields/methods would benefit from having the fandoc copied, ideally mapped, to jsdoc/typedoc